ARE_RestAPIlibraries/clientExample/sse.html

```diff
@@ -74,6 +74,7 @@
 			
 			function UNSUBSCRIBE_EVENTCHANNEL_EVENTS() {
 				var eventType = ServerEventTypes.EVENT_CHANNEL_TRANSMISSION;
+
```

Comment:
Was the empty line added on purpose? :slightly_smiling_face:

Reply:
no, I will checkout the the initial version